eras/alonzo/impl/src/Cardano/Ledger/Alonzo/Rules/Ledger.hs

```diff
@@ -101,14 +99,13 @@ ledgerTransition = do
             )
       else pure dpstate
 
-  let DPState dstate pstate = dpstate
+  let DPState dstate _pstate = dpstate
```

Comment:
Because the underscore `_` prefix means: do not produce a warning :wink: 

That is why it is bad to use record fields with underscores prefixes in them, like we had until recently in many types.